Calculator.Generic.INumber/RomanNumericsNumber.cs

```diff
@@ -19,6 +18,7 @@ public class RomanNumericsNumber : IAdditionOperators<RomanNumericsNumber, Roman
 
         });
         private int _numerics;
+        private string _numberRoman;
```

Comment:
this variable is never set, should be set in constructor when creating the instance.